tensorboard/webapp/feature_flag/views/types.ts

```diff
@@ -0,0 +1,32 @@
+/* Copyright 2022 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+import {FeatureFlags} from '../types';
+
+export const enum FeatureFlagStatus {
+  ENABLED = 'enabled',
+  DISABLED = 'disabled',
+  DEFAULT = 'default',
+}
+
+export type FeatureFlagState<K extends keyof FeatureFlags> = {
```

Comment:
"State" has a specific meaning in our application - it refers to data stored in ngrx. Can you rename?